app/models/season.rb

```diff
@@ -1,3 +1,5 @@
 class Season < ActiveRecord::Base
   validates :start_date, presence: true, uniqueness: true
+
+  has_many :weeks
```

Comment:
You can test this with shoulda_matchers like:

```ruby
it { should have_many(:weeks) }
```

More methods [here](https://github.com/thoughtbot/shoulda-matchers#activerecord-matchers)